dev/com.ibm.ws.jsp.2.3_fat/fat/src/com/ibm/ws/jsp23/fat/tests/JSPTests.java

```diff
@@ -745,6 +745,7 @@ public void testEL30MapCollectionObjectOperations() throws Exception {
      * @throws Exception
      *                       if something goes wrong
      */
+    @SkipForRepeat(SkipForRepeat.EE11_FEATURES) // ELContextImpl (within org.apache.jaspser.el) arguments changed changed in Pages 4.0
     @Test
     public void testJSP23ResolutionVariableProperties() throws Exception {
```

Comment:
Can you expand on this? What actually changed, do we have work to do to get this test working or is this a test you're expecting to skip?

Reply:
SkipForRepeat EE11 or later?